cdk/lib/app-config-stack.ts

```diff
@@ -0,0 +1,53 @@
+import * as cdk from '@aws-cdk/core';
```

Comment:
Let's add the shorter license header as in:

```javascript
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: MIT-0
```

Reply:
Added in e9d4fff